### PR TITLE
fix: icons cannot be dragged to the stash

### PR DESCRIPTION
**Pull request by xionglinlin**
1. Expanding the acceptance of drop
2. Fix drag and drop after opening the tray, the tray will close automatically

Issue: https://github.com/linuxdeepin/developer-center/issues/9675

### CODE REVIEW

File: panels/dock/tray/package/StashContainer.qml [hunk 1]
```diff
@@ -75,6 +75,18 @@ Item {
...
+            anchors.fill: parent
+            anchors.margins: itemSpacing
+
+            // Tray items
+            Repeater {
+                anchors.fill: parent
+                model: root.model
+                delegate: stashedItemDelegateChooser
+            }
+        }
```
Comment (BLumia) [inline panels/dock/tray/package/StashContainer.qml:89]: 这个 Item 需要放到 Rectangle 里吗？是不是放到它同层级的下面就行了

Comment (xionglinlin) [reply panels/dock/tray/package/StashContainer.qml]: 是的，这里不需要放到Rectangle，已经修改

**Review by BLumia — APPROVED** [review]

**Pull request merged**